tests/testthat/test-latest-dependencies.R

```diff
@@ -2,27 +2,39 @@ test_that("use_tidy_versions() specifies a version for dependencies", {
   skip_on_cran()
   withr::local_options(list(repos = c(CRAN = "https://cloud.r-project.org")))
 
-  pkg <- create_local_package()
+  create_local_package()
   use_package("usethis")
   use_package("desc")
-  use_package("withr", "Suggests")
-  use_package("gh", "Suggests")
   use_latest_dependencies()
-  desc <- read_utf8(proj_path("DESCRIPTION"))
-  desc <- grep("usethis|desc|withr|gh", desc, value = TRUE)
-  expect_true(all(grepl("\\(>= [0-9.]+\\)", desc)))
+
+  deps <- proj_deps()
+  expect_equal(
+    deps$version[deps$package %in% c("usethis", "desc")] == "*",
+    c(FALSE, FALSE)
+  )
+})
+
+test_that("use_tidy_versions() doesn't affect suggests", {
+  skip_on_cran()
+  withr::local_options(list(repos = c(CRAN = "https://cloud.r-project.org")))
+
+  create_local_package()
+  use_package("cli", "Suggests")
+
+  deps <- proj_deps()
+  expect_equal(deps$version[deps$package == "cli"], "*")
```

Comment:
I don't see a call to `use_tidy_versions()` (or `use_latest_dependencies()`) here.

Reply:
Ooops 😆